modules/service/src/main/scala/lucuma/odb/service/extensions.scala

```diff
@@ -23,3 +25,18 @@ class FromResultPartiallyApplied[F[_]]:
   def apply[A](result: Result[A])(using F: Applicative[F]): ResultT[F, A] =
     ResultT(F.pure(result))
 
+extension (self: SourceProfile)
+  def gaiaBands: Set[Band] =
```

Comment:
This is a constant. It can be confusing to have it as an instance method.

If it's meant to be accessed only from here maybe it can be a `private val GaiaBands` ?